R/zzz.R

```diff
@@ -10,6 +10,10 @@
 
   dm_has_financial <<- memoise::memoise(dm_has_financial, cache = cache_attach())
 
+  if (!identical(Sys.getenv("TESTTHAT"), "true")) {
+    check_suggested <<- memoise::memoise(check_suggested)
+  }
```

Comment:
It looks like the env variable "TESTTHAT" is set only at a later stage. The function is thus memoised even for the tests.

Reply:
Making it unconditional then.

Reply:
We can also define `check_suggested <- environment(check_suggested)$'_f'` in helper scripts, or keep an unmemoized copy in the namespace if we want to avoid using implementation details.

Reply:
Good idea, it seems that the tests work well enough even when always memoizing.